mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot_restore_job.go

```diff
@@ -58,16 +63,29 @@ func resourceMongoDBAtlasCloudProviderSnapshotRestoreJob() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
+						"oplog_ts": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"point_in_time_utc_seconds": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"oplog_inc": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
 					},
 				},
 				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
 					v := val.(map[string]interface{})
 
 					_, automated := v["automated"]
 					_, download := v["download"]
+					_, pointInTime := v["point_in_time"]
 
-					if (v["automated"] == "true" && v["download"] == "true") || (v["automated"] == "false" && v["download"] == "false") || (!automated && !download) {
-						errs = append(errs, fmt.Errorf("%q you need to implement only one: automated and download delivery types", key))
+					if (v["automated"] == "true" && v["download"] == "true" && v["point_in_time"] == "true") || (v["automated"] == "false" && v["download"] == "false" && v["point_in_time"] == "false") || (!automated && !download && !pointInTime) {
+						errs = append(errs, fmt.Errorf("%q you need to implement only one: automated or download or point_in_time delivery types", key))
```

Comment:
Just slight grammar edit
"you can only submit one type of restore job: automated, download or point_in_time

Reply:
done!